applications/GeoMechanicsApplication/custom_elements/U_Pw_small_strain_interface_element.cpp

```diff
@@ -1472,8 +1472,9 @@ void UPwSmallStrainInterfaceElement<3, 6>::CalculateShapeFunctionsGradients(TMat
     rAuxVariables.LocalCoordinatesGradientsMatrix(0, 1) = rAuxVariables.LocalCoordinatesGradients[0];
     rAuxVariables.LocalCoordinatesGradientsMatrix(1, 1) = rAuxVariables.LocalCoordinatesGradients[1];
 
-    GeoElementUtilities::InvertMatrix2(rAuxVariables.LocalCoordinatesGradientsInvMatrix,
-                                       rAuxVariables.LocalCoordinatesGradientsMatrix);
+    double determinant;
```

Comment:
I think it's always wise to initialize variables, even if you don't use them (like here). We could also make it more explicit that we don't need the determinant by expressing that in its name:
```suggestion
    auto unused_determinant = 0.0;
```
This suggestion also applies to changes at lines 1541 - 1543.